editor/layout/style.scss

```diff
@@ -0,0 +1,7 @@
+.editor-layout.sidebar-opened .editor-layout__content {
+	margin-right: $sidebar-width;
+
+	.editor-text-editor__formatting {
```

Comment:
We could also define this in `modes/text-editor/style.scss` as assuming the existence of the ancestor modifier:

```scss
.editor-text-editor__formatting {
	// ...

	.editor-layout.is-sidebar-opened & {
		// ...
	}
}
```